documentation/docs/verifiable_credentials/verifiable_presentations.mdx

```diff
@@ -10,17 +10,157 @@ keywords:
+
+The following properties may be specified on a presentation:
+
+- [**ID**](https://www.w3.org/TR/vc-data-model/#identifiers): optional URI identifier for the presentation.
+- [**Context**](https://www.w3.org/TR/vc-data-model/#contexts): list of JSON-LD context URIs. Includes `"https://www.w3.org/2018/credentials/v1"` by default.
+- [**Types**](https://www.w3.org/TR/vc-data-model/#types): list of types describing the presentation. Includes `"VerifiablePresentation"` by default.
+- [**Credentials**](https://www.w3.org/TR/vc-data-model/#dfn-verifiable-credentials): list of verifiable credentials to present.
+- [**Holder**](https://www.w3.org/TR/vc-data-model/#dfn-holders): optional URI, typically a DID, of the entity that generated the presentation.
+- [**Refresh Service**](https://www.w3.org/TR/vc-data-model/#refreshing): optional link to a service where the recipient may refresh the included credentials.
+- [**Terms of Use**](https://www.w3.org/TR/vc-data-model/#terms-of-use): optional list of policies defining obligations, prohibitions, or permissions of the presentation recipient.
+
+Of the above, only the list of credentials is required when creating a presentation using the framework.
+However, the holder property should be included to satisfy [subject-holder relationship](#subject-holder-relationship) checks during validation.
+
+After creation, the holder signs the verifiable presentation using a private key linked to one of the verification methods in their
+DID Document and transmits it to a verifier for [validation](#validation).
+
+### Proof Options {#proof-options}
+
+A digital signature on a verifiable presentation both provides data integrity and proves the DID of the holder.
+The proof section embedded in a presentation may also include additional metadata.
+
+The following metadata properties can be configured by the framework and are optional and omitted by default:
+
+- **Created**: timestamp of when the presentation was signed.
+- **Expires**: timestamp after which the presentation is no longer considered valid.
+- **Challenge**: arbitrary random string. Sent by the verifier and mitigates replay attacks; should be sufficiently random and uniquely generated per presentation request.
+- **Domain**: arbitrary string. Sent by the verifier and can help mitigate replay attacks when used with a challenge.
+- **Proof Purpose**: indicates the purpose of the signature.
+  - **AssertionMethod**: to assert a claim. The signing verification method must have an [`assertionMethod`](https://www.w3.org/TR/did-core/#assertion) relationship to be valid.
+  - **Authentication**: to authenticate the signer. The signing verification method must have an [`authentication`](https://www.w3.org/TR/did-core/#authentication) relationship to be valid.
+
+Verifiers should always send a challenge and domain to mitigate replay attacks, see [Security Considerations](#security-considerations).
```

Comment:
Should we emphasize this by either moving it up in the section or making it a ":::note"? Currently its not the emphasize of the section, but this is the most important aspect of it.

Reply:
Made it a note.